### ISSUE TITLE
Hi,I have a quetions about how to train my own model.

### ISSUE THREAD

**Issue by xvdehao** [davidsandberg/facenet#464]
Now I want to train a model by the program train_softmax.py. Then I encountered two question:
The first one: I try to run the program, but it cost me lots of time, so I changed the two factors(epoch_size,epoch_size).Then I can get a result,but I cant understant what the two factors mean.
Here is my whole code:
`python src/train_softmax.py --logs_base_dir ~/logs/facenet/ --models_base_dir ~/models/facenet/ --data_dir /opt/hand_test/CASIA-PalmprintV1/ --image_size 128 --model_def models.inception_resnet_v1 --optimizer RMSPROP --learning_rate -1 --max_nrof_epochs 1 --keep_probability 0.8 --random_crop --random_flip --learning_rate_schedule_file data/learning_rate_schedule_classifier_casia.txt --weight_decay 5e-5 --center_loss_factor 1e-2 --center_loss_alfa 0.8 --epoch_size 50`

The second one:I  cant get the .pb file. What is the .pb file mean? Why I cannt get the .pb file?

**Comment by MaartenBloemen** [comment]
Hi!
So regarding you first question, it is normal that the training takes a lot of time. It all depends on the hardware you use, for me training on Casia for example takes around 10 hours.
For your second question, a .pb file is a protobuf file and is just a frozen graph. You can get a protobuf file by using [freeze_graph.py](https://github.com/davidsandberg/facenet/blob/master/src/freeze_graph.py).
I hope this clears things up for you!

**Comment by xvdehao** [comment]
These are really helpful to me.
Alright,maybe I need more time then you with a virtual machine.
And I have another question:when I want to verify the face by my own trained model, I can only get the poor accuracy, maybe 0.3~0.4,Why its too small? Do I have some  methods to increase it ?
(FYI,I used 10 calsses, about 50 imagec)

**Comment by MaartenBloemen** [comment]
I don't really know why you would get low accuracy, I use about 20 images per class and I get around 75-80% accuracy, so it's quite weird you only get such a low accuracy...

**Comment by xvdehao** [comment]
ha maybe something is wrong
But 75~85% is also not a satisfied accuracy? 
So it depends on the images per class?
Maybe I can try it, then do you have any other suggestions? 
Looking forward your reply,thanks
